solutions/devicesimulation/single-vm/docker-compose.yml

```diff
@@ -101,8 +101,11 @@ services:
       - PCS_AUTH_AUDIENCE
       - PCS_AUTH_REQUIRED
       - PCS_CORS_WHITELIST
+      - PCS_CLOUD_TYPE
+      - PCS_SUBSCRIPTION_ID
       - PCS_DEPLOYMENT_ID
       - PCS_SOLUTION_TYPE
+      - PCS_IOTHUB_NAME
       - PCS_DIAGNOSTICS_ENDPOINT_URL
```

Comment:
can you also add PCS_DIAGNOSTICS_ENDPOINT_URL as a var for telemetry microservice?

Reply:
yes, I will add it.

Reply:
@molly-moen , is it only required for telemetry service? what about the other services?